modules/monitoring/cmk.tf

```diff
@@ -0,0 +1,65 @@
+# Creates/manages KMS CMK
+resource "aws_kms_key" "cmk" {
+  count = var.enable_cmk ? 1 : 0
+
+  #description             = var.description
+  customer_master_key_spec = var.key_spec
+  is_enabled               = var.enabled
+  enable_key_rotation      = var.rotation_enabled
+  #tags                    = var.tags
+  deletion_window_in_days = 30
+
+  policy = jsonencode({
+    "Version" : "2012-10-17",
+    "Id" : "key-default-1",
```

Comment:
Are you sure that this ID should be named key-default-1?

Reply:
changed to 'kms-key-policy-access-control'